lib/ibm_speech_to_text.ex

```diff
@@ -65,32 +64,32 @@ defmodule Membrane.Element.IBMSpeechToText do
         Keyword.merge(rec_opts, content_type: :flac, inactivity_timeout: -1)
       end)
 
-    {:ok, state}
+    {[], state}
   end
 
   @impl true
-  def handle_prepared_to_playing(_ctx, state) do
+  def handle_playing(_ctx, state) do
     with {:ok, pid} <- Client.start_link(state.region, state.api_key, state.client_options) do
       Membrane.Logger.info("IBM API Client started")
-      {{:ok, demand: :input}, %{state | connection: pid}}
+      {[demand: :input], %{state | connection: pid}}
     end
   end
 
   @impl true
-  def handle_playing_to_prepared(_ctx, %{connection: conn} = state) do
+  def handle_terminate_request(_ctx, %{connection: conn} = state) do
     Client.stop(conn)
     Membrane.Logger.info("IBM API Client stopped")
-    {:ok, %{state | connection: nil}}
+    {[terminate: :normal], %{state | connection: nil}}
   end
```

Comment:
If we start connection with UtilitySupervisor we probably can delete this callback?